src/components/file.js

```diff
@@ -107,6 +108,12 @@ module.exports = function(app) {
           Formio
         ) {
           if ($scope.options && $scope.options.building) return;
```

Comment:
We won't be able to commit this because this will end up clobbering existing files within S3.